deployment/stacks/service/base/template/kustomization.yaml

```diff
@@ -25,7 +25,7 @@ configMapGenerator:
       - DB_DOS_USERNAME=DB_DOS_USERNAME_TO_REPLACE
       - DB_DOS_PASSWORD=DB_DOS_PASSWORD_TO_REPLACE
       - API_LOG_LEVEL=API_LOG_LEVEL_TO_REPLACE
-      - APP_ADMIN_PASSWORD=APP_ADMIN_PASSWORD_TO_REPLACE
+      - API_ADMIN_PASSWORD=API_ADMIN_PASSWORD_TO_REPLACE
```

Comment:
Secret Store will need to be updated to reflect the name change here when merged in. Left this comment as a reminder to update across all environments

Reply:
The Secret Store variable name changed.